firstvoices/backend/search/utils/validators.py

```diff
@@ -148,3 +150,23 @@ def get_valid_site_feature(input_site_feature_str):
     if len(selected_values) == 0:
         return None
     return selected_values
+
+
+def get_valid_site_slugs(input_site_slug_str, user):
```

Comment:
Since we are already making a db query here, why not return the valid site IDs instead of the slugs? It would avoid needing to index the site slug in every index. This function could be get_valid_site_ids_from_slugs